runtime/tests/TestUtils.hpp

```diff
@@ -24,18 +24,35 @@
 
 #include "catch2/catch.hpp"
 
-#include "LightningSimulator.hpp"
+#include "ExecutionContext.hpp"
+#include "QuantumDevice.hpp"
+
+#include <iostream>
+
+/// @cond DEV
+namespace {
+using namespace Catalyst::Runtime;
+} // namespace
+/// @endcond
 
 /**
  * A tuple of available backend devices to be tested using TEMPLATE_LIST_TEST_CASE in Catch2
  */
+#if __has_include("LightningSimulator.hpp")
+#include "LightningSimulator.hpp"
 #if __has_include("LightningKokkosSimulator.hpp")
 #include "LightningKokkosSimulator.hpp"
 using SimTypes = std::tuple<Catalyst::Runtime::Simulator::LightningSimulator,
                             Catalyst::Runtime::Simulator::LightningKokkosSimulator>;
 #else
 using SimTypes = std::tuple<Catalyst::Runtime::Simulator::LightningSimulator>;
 #endif
+#else
+#if __has_include("LightningKokkosSimulator.hpp")
+#include "LightningKokkosSimulator.hpp"
+using SimTypes = std::tuple<Catalyst::Runtime::Simulator::LightningKokkosSimulator>;
```

Comment:
Is this still relevant with kokkos gone?

Reply:
removed!